CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+This template and our [recommendation pages][website] were made by [RSE]s at
+[UCL's Centre for Advanced Research Computing][UCL ARC]. We made it with
+Research Software projects in mind, but whoever you are, we hope you find this
+useful!
+
+We are actively encouraging users to ask questions and start discussions in the
+[discussions tab] of this repository. Does something seem like it's broken?
+Please go ahead and [open an issue]!
+
+The [website] pages are open to contributions but they will need to be reviewed
+by a member or associate member of ARC. We might be slow to approve new tool
+suggestions (since we'll probably want to discuss them first) but don't let that
+put you off creating an issue.
+
+## Development workflow
+
+To contribute a change, please:
+
+1. Create a new branch (or fork if you're not in ARC-collaborations).
```

Comment:
Do we mean the `Collaborations` GitHub team in the `UCL-ARC` organization here (as assuming this was given default write permissions)? Not sure if GitHub teams pages are visible outside of organization but could possibly link to https://github.com/orgs/UCL-ARC/teams/collaborations to make this explicit / easy for people to check, if so.

Reply:
- [ ] do this